p4sdk/images/Dockerfile

```diff
@@ -13,13 +13,15 @@ WORKDIR /src
 RUN mkdir -pv /opt/p4 && tar -zxvf p4.tar.gz -C /opt/p4 && rm -f p4.tar.gz
 
 FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b
-RUN apk add --no-cache --virtual gawk gettext
-COPY --from=builder /opt/p4 /opt/p4
+RUN apk add --no-cache --virtual openssl libressl-dev ncurses-libs libstdc++ gawk gettext
+COPY --from=builder /opt/p4 /opt/
 COPY p4 /root/linux_networking
 COPY images/entrypoint.sh /
+COPY images/es2k_skip_p4.conf.template /
 
 LABEL io.k8s.display-name="IPU P4 SDK"
 
 RUN rm -rf /var/cache/apk/*
 WORKDIR /
 ENTRYPOINT ["/entrypoint.sh"]
+# ENTRYPOINT ["/bin/sh","-c","while true; do sleep 3000; done"]
```

Comment:
remove

Reply:
we're keeping this for quick local testing for now